st2client/st2client/formatters/execution.py

```diff
@@ -29,6 +30,8 @@
 
 LOG = logging.getLogger(__name__)
 
+PLATFORM_MAXINT = 2 ** (struct.Struct('i').size * 8 - 1) - 1
```

Comment:
Python 3's int is just the original `long` type in 2, which has a near-infinite length. This could cause issues

Reply:
This returns maxint as by the underlying C library so it should be fine aka the same as `sys.maxint` we used in the past.